src/vos/tests/SConscript

```diff
@@ -32,10 +32,9 @@ def scons():
 
     libraries = ['daos_common_pmem', 'daos_tests', 'gurt', 'cart', 'cmocka',
                  'vos', 'uuid', 'pmem', 'pmemobj', 'bio', 'pthread', 'abt']
-    test = unit_env.d_test_program('pool_scrubbing_tests',
-                                   ['pool_scrubbing_tests.c', '../vos_pool_scrub.c'],
+    unit_env.d_test_program('pool_scrubbing_tests', ['pool_scrubbing_tests.c',
+                                                     '../vos_pool_scrub.c'],
                                    LIBS=libraries)
-    unit_env.Install('$PREFIX/bin/', test)
```

Comment:
LIBS=libraries should line up with 'pool_scrubbing_tests'...looks like it's off according to the flake message.

Reply:
Think I've fixed this.